crt_portal/cts_forms/views.py

```diff
@@ -916,6 +916,21 @@ def _notification_change(request):
     return redirect(reverse('crt_forms:crt-forms-notifications'))
 
 
+@login_required
+def resources_view(request):
+    resources = Resource.objects.all()
+    data = []
+    for index, resource in enumerate(resources):
+        data.append({
+            'resource': resource,
+            'tags': list(map(str, resource.tags.values_list('name', flat=True))),
+        })
+    data_dict = {
+        'data_dict': data,
+    }
+    return render(request, 'forms/complaint_view/resources/index.html', data_dict)
```

Comment:
nit: naming - this isn't actually a dict, it's a list

Maybe just call this `resources` or `table_entries` to give it a bit of context?